src/engine/HedronEngine.ts

```diff
@@ -95,7 +95,7 @@ export class HedronEngine {
       const sketchInstances = this.sketchManager!.getSketchInstances()
       debugScene.clearPasses()
       Object.values(sketches).forEach((sketch) => {
-        // eslint-disable-next-line
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
         const paramValues: { [key: string]: any } = {}
```

Comment:
Maybe it would be good to get this typed, but I can forsee wanting to have objects like a vec2, vec3, gradient, in the future. Maybe even want to support anything via plugins, and once it's external peoples desires may be endless